cmd/lockrelease/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 The Kubernetes Authors.
```

Comment:
> Noted that, since the lock release controller is now an individual pod, we need to update our Louhi workflow to build the image for the lock release controller and modify auto-vc for automatic CVE fixes. We should also create a ticket as a reminder.

Created a ticket to keep track b/372777238